codegen/src/main/scala/com/soundcloud/twinagle/codegen/Twinagle.scala

```diff
@@ -16,9 +17,8 @@ object Twinagle extends AutoPlugin {
       Target(scalapb.gen(grpc = false), (sourceManaged in Compile).value),
       Target(JvmGenerator("scala-twinagle", ServerClientCodeGenerator), (sourceManaged in Compile).value)
     ),
-    libraryDependencies ++= List(
-      "com.soundcloud" %% "twinagle-runtime" % com.soundcloud.twinagle.codegen.BuildInfo.version,
-      "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion
+    libraryDependencies ++= Seq(
+      "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion % "protobuf"
```

Comment:
so this wraps the well-known types?

Reply:
kinda. `scalapb-runtime` [depends](http://central.maven.org/maven2/com/thesamet/scalapb/scalapb-runtime_2.12/0.9.0-RC1/scalapb-runtime_2.12-0.9.0-RC1.pom) on `protobuf-java`, which contains the `.proto` definitions. scalapb unzips them from the jar and hands them to protoc to do the actual code generation.